### PR TITLE
Unit testing

### PR DESCRIPTION
**Pull request by csala**

### CODE REVIEW

File: tests/btb/test_hyperparameter_old.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import copy
```
Comment (lauragustafson) [inline tests/btb/test_hyperparameter_old.py]: @csala I see that you renamed test_hyperparameter.py to test_hyperparameter_old.py. Is there a new set of unit tests for the HyperParameter classses? If so I don't see them in the PR.

File: btb/tuning/gp.py [hunk 2]
```diff
@@ -32,8 +33,9 @@ def fit(self, X, y):
             return
 
         # old gaussian process code
-        #self.gp = GaussianProcess(theta0=1e-2, thetaL=1e-4, thetaU=1e-1,
-        #                          nugget=np.finfo(np.double).eps * 1000)
+        # TODO: Can this be removed?
```
Comment (lauragustafson) [inline btb/tuning/gp.py]: Yes I believe so.

File: btb/tuning/gp.py [hunk 3]
```diff
@@ -42,8 +44,10 @@ def predict(self, X):
             # we probably don't have enough
             logger.warn('GP: not enough data, falling back to uniform sampler')
             return Uniform(self.tunables).predict(X)
+
         # old gaussian process code
-        #return self.gp.predict(X, eval_MSE=True)
+        # TODO: Can this be removed?
```
Comment (lauragustafson) [inline btb/tuning/gp.py]: Yes

File: examples/rosenbrock.py [hunk 4]
```diff
@@ -19,8 +19,8 @@ def rosenbrok(x, y):
 
 
 def find_min_with_tuner(tuner):
-    minimum_score = float("inf")
-    xy_min = None
+    # minimum_score = float("inf")
```
Comment (lauragustafson) [inline examples/rosenbrock.py]: If you want, you can just remove these lines.

File: tests/btb/test_hyperparameter_old.py (hunk 1 above)
Comment (csala) [reply tests/btb/test_hyperparameter_old.py]: No! I actually renamed it to `_old` before creating the new test files but at the end decided to not rewrite the ones for this module and forget about it. I'll rename it.

**Pull request merged**